src/link_manager/LinkManagerStateMachineActiveStandby.cpp

```diff
@@ -426,6 +426,8 @@ void ActiveStandbyStateMachine::activateStateMachine()
         mStartProbingFnPtr();
 
         updateMuxLinkmgrState();
+
+        mMuxPortPtr->checkWarmRestart();
```

Comment:
In this design won't the reconciliation timer start only when all ports are anyway reconciled? Shouldn't the timer start before ` mMuxPortPtr->checkWarmRestart();` is called (when the components are getting initialized)?

Reply:
Yes `checkWarmRestart` is called after port is reconciled, but timer has been started before any port is reconciled. 

`checkWarmRestart` is to check if
* need to set port back to `auto` mode
* timer can be cancelled if this is the last port